selftests/unit/test_safeloader.py

```diff
@@ -47,6 +47,21 @@ def test_import_inside_class(self):
 
 class DocstringTag(unittest.TestCase):
 
+    NO_TAGS = [":AVOCADO: TAGS:FAST",
+               ":AVOCADO: TAGS=FAST",
+               ":avocado: mytags=fast",
+               ":avocado: tags",
+               ":avocado: tag",
+               ":avocado: tag="]
```

Comment:
another invalid tag would be `:this is not :avocado: tags=foo` or `tags=bar`... anyway you already cover quite a lot...